crates/flux-middle/src/global_env.rs

```diff
@@ -591,6 +596,14 @@ impl<'genv, 'tcx> Map<'genv, 'tcx> {
         }
     }
 
+    pub fn fn_reveals_for(self, def_id: LocalDefId) -> QueryResult<&'genv [FluxDefId]> {
```

Comment:
You are not accessing this function outside.

```suggestion
    fn fn_reveals_for(self, def_id: LocalDefId) -> QueryResult<&'genv [FluxDefId]> {
```

Reply:
It is used in `define_funs` inside `fixpoint_encoding` (to see if a function should be `define` or `constant` ...)